src/Microsoft.AspNetCore.Build/project.json

```diff
@@ -1,26 +1,31 @@
 ﻿{
-  "version": "1.0.0-*",
+    "version": "1.0.0-*",
 
-  "buildOptions": {
-    "emitEntryPoint": true
-  },
+    "buildOptions": {
+        "emitEntryPoint": true
+    },
 
   "publishOptions": {
     "include": [ "Microsoft.AspNetCore.Build.tasks" ]
   },
 
-  "dependencies": {
-    "Microsoft.NETCore.App": {
-      "version": "1.0.0-*",
-      "type": "platform"
+    "dependencies": {
+        "Microsoft.NETCore.App": {
+            "version": "1.0.0-*",
```

Comment:
Yeah, it's only there because we have to make this a console app because we need to "publish" it due to limitations in MSBuild Core, but I agree it'll be better to lock it.